{"uuid":"29828c80-3b50-4e9c-85d2-e20abe032ad4","name":"Scenario Outline: The user is presented with error message for code with invalid syntax in tryEditor for Arrays in Python page","children":["ed91840d-c011-4526-9920-3e0c80b8deb7"],"befores":[],"afters":[{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[{"name":"My screenshot","source":"ea633380-11e2-466b-940f-c3417706216b-attachment","type":"image/png"},{"name":"Myscreenshot","source":"aba36847-5371-4ba2-a306-627de8ccce38-attachment"}],"parameters":[],"start":1696885537165,"stop":1696885537538},{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"skipped","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"AFTER_STEP is failed."},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696885537539,"stop":1696885537539},{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"skipped","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"AFTER_STEP is failed."},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696885537541,"stop":1696885537542},{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"skipped","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"AFTER_STEP is failed."},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696885537545,"stop":1696885537545}],"start":1696885537138,"stop":1696885537546}